src/pokeAPI.js

```diff
@@ -0,0 +1,34 @@
+const search_term = document.getElementById('search_q')
+const search_btn = document.getElementById('search-btn')
+
```

Comment:
- [ ] We don't need this search functionally because we will just display data on the page.